modules/security/mtls/README.md

```diff
@@ -1,94 +1,119 @@
 Establish mTLS throughout the mesh
 ====
 
-Enable mTLS for service-to-service
-----
+By default, Istio's mesh is automatically configure as `permissive`, which means Istio will use mTLS between workloads with a Sidecar and will revert to plain protocol (plaintext HTTP for example) if not. This is a great way to onboard new clusters and workload.
+Still, at some point, you're going to need to limit the non mTLS traffic.
```

Comment:
```suggestion
Permissive mode is not really secure because any client can request to downgrade to clear text - it exists only to help transition your mesh into mTLS without breaking existing applications. At some point, you're going to need to limit non-mTLS traffic.
```